docs/architecture/adr-005-light-client-fork-detection.md

```diff
@@ -45,3 +45,47 @@ one correct full node in order to detect conflicts in a timely fashion. We keep
 this mechanism simple for now, but in the future a more advanced peer discovery
 mechanism may be utilized.
 
+
+#### Sequential Sync
```

Comment:
Should this be split into it's own PR?

Reply:
Sure! It was a separate PR (see #148) but can definitely be broken out again if it makes sense.